apps/storybook/src/stories/datacom-loader.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useEffect } from 'react';
+import { StoryFn } from '@storybook/react';
+import { DatacomLoader, DatacomButton } from '@datacom/endeavour-react';
+import styled from '@emotion/styled';
+
+type LoaderProps = React.ComponentProps<typeof DatacomLoader>;
+
+export default {
+  title: 'Loader',
+  component: DatacomLoader,
+  argTypes: {
+    label: {
+      name: 'Label',
+      defaultValue: 'Loading message',
+      description: 'Label for loading status.',
+      type: { name: 'string' },
+    },
+    loadingStatus: {
+      name: 'Loading Status',
+      defaultValue: 'default',
+      control: 'select',
+      description: 'Loading status is default if it is not set',
+      options: ['default', 'none', 'error', 'success'],
+    },
+  },
+};
+
+const Template: StoryFn<LoaderProps> = (args) => {
```

Comment:
This template isn't being used. Either remove it or use it in one of the stories below.